typescript/big-practice/src/scripts/controllers/noteController.ts

```diff
@@ -35,47 +35,41 @@ export default class NoteController {
 
   renderTabs() {
     const handlers = {
-      renderTabNotes: () => this.renderTabNote(),
-      renderTabTrash: () => this.renderTabTrash(),
+      renderTabNotes: () => this.renderTab('listNotes'),
+      renderTabTrash: () => this.renderTab('trashNotes'),
       addNote: (note: Note) => this.addNote(note),
     };
 
     this.view.renderTabs(handlers);
   }
 
-  async renderTabTrash() {
+  async renderTab(tab = '') {
```

Comment:
Why should we add param like that?

Reply:
Don't need a pass param, you can use tab?:string instead of